test_outputs/function_missing.output.ron

```diff
@@ -1,4 +1,16 @@
 {
+    "./test_crates/features_simple/": [
+        {
+            "name": String("feature_dependant_function"),
```

Comment:
typo: `dependant` -> `dependent` -- can you rename the directory and update all of these accordingly?

Reply:
You meant the function, right?

Reply:
I was worried it might also be a crate name, but if it's just one function then even better :)